cornflow/endpoints/dag.py

```diff
@@ -38,6 +38,9 @@ def post(self, idx):
         # because we do not want to store airflow's user:
         req_data['user_id'] = execution.user_id
         req_data['finished'] = True
+        # TODO: here I guess that the run is performed without problems so state should be 1
+        req_data['state'] = 1
```

Comment:
We should define the constants in some other file, and use them here, such as EXEC_STATE_CORRECT=1, EXEC_STATE_ERROR=-1, EXEC_STATE_RUNNING=0, EXEC_STATE_STOPPED=-2, EXEC_STATE_UNKNOWN=-3 etc.

Reply:
Ok, I can do that, but for now, the state can only take the following values. -1, 0 and 1. I have set up a constraint in the schema valdiation and in the database, if want to have more codes I have to change also those constraints